<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookRanking</class>
 <widget class="QWidget" name="BookRanking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_popularborrow">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>热门借阅榜</string>
   </property>
  </widget>
  <widget class="Line" name="line_popular">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>1230</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lb_categoryrank">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>分类排行榜</string>
   </property>
  </widget>
  <widget class="Line" name="line_classify">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>1230</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QFrame" name="fm_collect">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>230</width>
     <height>290</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lb_collect">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>收藏榜</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_collect1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>165</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>史记</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_collect2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>165</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>南京大屠杀</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_collect3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>165</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>银河帝国：基地七步曲</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="fm_man">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>230</width>
     <height>290</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lb_man">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>男生榜单</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_man3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>165</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>银河帝国：基地七步曲</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_man1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>165</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>史记</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_man2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>165</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>南京大屠杀</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="fm_woman">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>410</y>
     <width>230</width>
     <height>290</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lb_woman">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>女生榜单</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_woman3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>165</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>银河帝国：基地七步曲</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_woman2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>165</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>南京大屠杀</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_woman1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>165</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>史记</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="fm_newbook">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>410</y>
     <width>230</width>
     <height>290</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lb_newbook">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>新书推荐</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_new3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>165</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>银河帝国：基地七步曲</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_new1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>165</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>史记</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_new2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>165</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>南京大屠杀</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_book1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>130</width>
     <height>170</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookname1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="lb_book2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>130</width>
     <height>170</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookname2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookname3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_book3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>130</width>
     <height>170</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_book4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>130</width>
     <height>170</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookname4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookname5">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>260</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_book5">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>80</y>
     <width>130</width>
     <height>170</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookname6">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>260</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_book6">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>80</y>
     <width>130</width>
     <height>170</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookname7">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>260</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_book7">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>80</y>
     <width>130</width>
     <height>170</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
